python/lsst/ts/ess/common/sensor/boltek_ld250.py

```diff
@@ -101,6 +102,15 @@ class Ld250Sensor(BaseSensor):
     value for int and a zero-padded value for float values.
     """
 
+    def __init__(
+        self,
+        log: logging.Logger,
+    ) -> None:
+        super().__init__(log=log)
+
+        # Override default value.
+        self.charset = "UTF-8"
```

Comment:
Same comment as above.

And you might consider making "utf-8" the default, since that is so common in Python, with other values being overrides. But that depends, I guess, on how many subclasses you have, and what other values they need.

Another option is not default, but that gets a bit trickier with class variables (detecting the absence is a bit more work). If it's an instance variable you can just make it a required constructor argument. Still...it really feels like a class variable to me, so I think it's worth the work.

Reply:
Actually making utf-8 the default is probably a bad idea if the device only outputs ASCII. And if a device is using serial then I suppose the odds of it outputting utf-8 are pretty small.

Reply:
Reverted to ASCII.